manifests/r/RoyalApps/RoyalTS/6.00.50623.0/RoyalApps.RoyalTS.yaml

```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.00.50623.0
```

Comment:
```suggestion
PackageVersion: 6.0.50623.0
```

Reply:
Hi @ItzLevvie, should i change the folder name as well?